.gitignore

```diff
@@ -29,3 +29,10 @@ enclave/enclave
 # Ignore ISLET-related download and build artifacts
 *shim_test
 *.tar.gz
+
+# Ignore files generated by Python bindings' build process(es)
+__pycache__
+certifier_framework_wrap.cc
+certifier_tests_wrap.cc
+libcertifier_framework.py
+libcertifier_tests.py
```

Comment:
All generated files. I think these need not be committed.

We can change our minds once we get better experience with the build processes and toolchain, once simple-app in Py gets developed.